generated/__init__.py

```diff
@@ -0,0 +1,13 @@
+from nidaqmx.errors import DaqError, DaqReadError, DaqWriteError, DaqWarning, DaqResourceWarning
```

Comment:
Please move the files to `generated/nidaqmx/` so that the `generated/` directory contains a fully-formed `nidaqmx` module. We don't want to make `generated` part of the module hierarchy.

For comparison, nimi-python modules live in `generated/${module}/${module}/` like [generated/nidcpower/nidcpower/](https://github.com/ni/nimi-python/blob/master/generated/nidcpower/nidcpower/). The extra directory level is necessary to give each nimi-python module a separate `setup.py`. There is only one `nidaqmx` module in this repo, so this repo doesn't need the extra directory level.

Reply:
Moved the generated files to `generated/nidaqmx`